async/amqp/amqp.go

```diff
@@ -32,6 +32,7 @@ type message struct {
 	del     *amqp.Delivery
 	dec     encoding.DecodeRawFunc
 	requeue bool
+	queue   string
```

Comment:
maybe name this also source like the getter. Queue is more message broker specific, Kafka has topics,..